GitCommands/Git/GitModule.cs

```diff
@@ -3444,6 +3444,15 @@ public IEnumerable<string> GetPreviousCommitMessages(int count, string revision
                 });
         }
 
+        // Known bug on git-for-windows: Comparing working directory to any branch, fails, due to -R
+        // I.e., git difftool --gui --no-prompt --dir-diff -R HEAD fails, but
```

Comment:
If so, information about the diff need to be transferred to _revisionDiffProvider.Get()
The -R option is used to display diffs correctly when comparing staged/unstaged artificial commits to other commits.

Reply:
First - I am but one person using this application. I did not know if this "known bug" is actually a bug for all, or a bug for me. I'd appreciate help testing this.

Second, I understand the need for the -R, but I was unsure as to what to do in this case, if the git-for-windows bug is truly broken for everyone. I could add logic to _revisionDiffProvider to know whether or not it is a dir-diff, and open a backwards diff. Or fall back to the GitExtensions display, file by file.